auth-web/src/components/auth/AddNameRequestForm.vue

```diff
@@ -0,0 +1,247 @@
+<template>
+  <div class="passcode-form">
+    <v-form ref="addNRForm" lazy-validation>
+      <fieldset>
+        <v-expand-transition>
+          <div class="passcode-form__alert-container" v-show="validationError">
+            <v-alert
+              :value="true"
+              color="error"
+              icon="warning"
+            >{{validationError}}
+            </v-alert>
+          </div>
+        </v-expand-transition>
+          <v-text-field
+            filled
+            label="Enter a Name Request Number"
+            hint="Example: NR 1234567"
+            req
+            persistent-hint
+            :rules="entityNumRules"
+            v-model="nameRequestNumber"
+            @blur="incorpNumFormat"
+            data-test="business-identifier"
+          ></v-text-field>
+          <v-text-field
+            filled
+            label="Enter the Applicant Phone Number"
+            hint="Example: 555-555-5555"
+            persistent-hint
+            :rules="entityPhoneNumberRules"
+            v-model="applicantPhoneNumber"
+            type="tel"
+            v-mask="['###-###-####']"
+            data-test="entity-phonenumber"
+          ></v-text-field>
+          <div class="font-weight-bold ml-3 mb-2">or</div>
+          <v-text-field
+            filled
+            label="Enter the Applicant Email Address"
+            hint="Example: name@email.com"
+            persistent-hint
+            :rules="entityEmailRules"
+            v-model="applicantEmail"
+            data-test="entity-email"
+          ></v-text-field>
+      </fieldset>
+      <div class="form__btns mt-8">
+        <v-btn large text class="pl-2 pr-2 lost-passcode-btn" data-test="forgot-passcode-button" @click.stop="helpDialog = true">
+          <v-icon>mdi-help-circle-outline</v-icon>
+          <span>I lost or forgot my Name Request (NR) Number</span>
+        </v-btn>
+        <v-btn
+          data-test="add-business-button"
+          large color="primary"
+          :disabled="!isFormValid()"
+          @click="add"
+          :loading="isLoading"
+        >
+          <span>Add</span>
+        </v-btn>
+        <v-btn large depressed color="default" class="ml-2" data-test="cancel-button" @click="cancel">
+          <span>Cancel</span>
+        </v-btn>
+      </div>
+    </v-form>
+    <v-dialog v-model="helpDialog" max-width="640">
+      <v-card>
+        <v-card-title>Need Assistance?</v-card-title>
+        <v-card-text>
+          <p class="mb-7">If you have not received your Access Letter from BC Registries, or have lost your Name Request (NR) Number, please contact us at:</p>
```

Comment:
does it make sense to separate this out as a separate component?

Reply:
Yes, was planning to do that after the main tasks